tests/phpunit/tests/test-preload-paths.php

```diff
@@ -117,4 +117,27 @@ public function test_search_rest_route_is_filterable() {
 		$this->assertCount( 1, $preload_paths );
 		$this->assertSame( '/wp/v2/search?test=something', reset( $preload_paths ) );
 	}
+
+	/**
+	 * @ticket #2211 {@see https://github.com/polylang/polylang-pro/issues/2211}
+	 */
+	public function test_navigation_fallback_rest_route_is_filterable() {
+		add_filter(
+			'pll_get_post_types',
+			function ( $post_types ) {
+				return array_merge(
+					$post_types,
+					array( 'wp_navigation' => 'wp_navigation' )
+				);
+			}
+		);
+		$this->pll_admin->model->cache->clean( 'post_types' );
+		$this->pll_admin->block_editor->filter_rest_routes = new PLL_Filter_REST_Routes( $this->pll_admin->model );
```

Comment:
Why do you have to re init `PLL_Filter_REST_Routes` ?

Reply:
Because the routes are initialized in its constructor, and the test momentary adds `wp_navigation` to translated post types.

Reply:
We should not need to manually add `wp_navigation` as it's always translated. Would the test work as expected by instantiating `PLL_Sync_Navigation`?

Reply:
This is the case in Polylang Pro, not Free.